docs/syzbot.md

```diff
@@ -98,7 +98,7 @@ parallel).
 A syzkaller program can be converted to an almost equivalent C source using `syz-prog2c` utility. `syz-prog2c` has lots of flags in common with [syz-execprog](https://github.com/google/syzkaller/blob/master/docs/executing_syzkaller_programs.md), e.g. `-threaded`/`-collide` which control if the syscalls are executed sequentially or in parallel. An example invocation:
 
 ```
-syz-prog2c -prog repro.syz.txt -threaded -collide -repeat -procs=8 -sandbox=namespace -tun -tmpdir -waitrepeat
+syz-prog2c -prog repro.syz.txt -enable=all -threaded -collide -repeat -procs=8 -sandbox=namespace -segv -tmpdir -waitrepeat
```

Comment:
How did you decide what goes into enable/disable and what stays as flags? threaded/collide/segv/tmpdir look like the other things that you moved to enable/disable, yet they stay as flags.

Reply:
I've used the `enable/disable` for the features that enable or disable fuzzing surfaces. Like when `tun` is enabled we fuzz the networking stack externally or with `cgroups` we fuzz cgroups related stuff. `threaded/collide/segv/tmpdir` seem to be more unique and have to do with how we fuzz, rather than what we fuzz.